File: csharp/1046.cs
using System; 

class URI {

    static void Main(string[] args) { 

        string[] entrada = Console.ReadLine().Split(' ');
        int h1 = int.Parse(entrada[0]), h2 = int.Parse(entrada[1]), resultado = 0;
        if (h1 > h2)
        {
            resultado = (24-h1)+h2;
            Console.WriteLine($"O JOGO DUROU {resultado} HORA(S)");
        }
        else if (h1 < h2)
        {
            resultado = h2 - h1;
            Console.WriteLine($"O JOGO DUROU {resultado} HORA(S)");
        }
        else
        {
            resultado = (24-h1)+h2;
            Console.WriteLine($"O JOGO DUROU 24 HORA(S)");
        }

    }

}

File: java 19/1046.java
import java.io.IOException;
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);        
        int n1 = sc.nextInt();
        int n2 = sc.nextInt();
        int resultado = 0;
        if (n1 < n2){
            resultado = (n2-n1);
        } else if (n1 == n2){
            resultado = (24);
        } else {
            resultado = ((24-n1)+n2);
        }
        System.out.printf("O JOGO DUROU %d HORA(S)\n",resultado);
        sc.close();
    }
}
